GameServerLib/Logic/Game.cs

```diff
@@ -47,28 +48,24 @@ public class Game
         protected const double REFRESH_RATE = 1000.0 / 30.0; // 30 fps
         private Logger _logger;
         // Object managers
-        private ItemManager _itemManager;
+        private readonly ItemManager _itemManager;
         // Other managers
-        private ChatCommandManager _chatCommandManager;
-        private PlayerManager _playerManager;
-        private NetworkIdManager _networkIdManager;
+        private readonly ChatCommandManager _chatCommandManager;
+        private readonly PlayerManager _playerManager;
+        private readonly NetworkIdManager _networkIdManager;
+        private readonly IPacketHandlerProvider _packetHandlerProvider;
         private Stopwatch _lastMapDurationWatch;
 
         private List<GameScriptTimer> _gameScriptTimers;
 
-        public Game(
-            ItemManager itemManager,
-            ChatCommandManager chatCommandManager,
-            NetworkIdManager networkIdManager,
-            PlayerManager playerManager,
-            Logger logger
-        )
+        public Game(ItemManager itemManager, ChatCommandManager chatCommandManager, NetworkIdManager networkIdManager, PlayerManager playerManager, Logger logger, IPacketHandlerProvider handlersProvider)
```

Comment:
This should be expanded over several lines to follow our guideline of 120 characters per line maximum. Optional solution would be to create a `GameParams` struct and take in a single argument of that type instead.